manifests/p/Postman/Postman/10.9.4/Postman.Postman.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 10.9.4
+PackageLocale: en-US
+Publisher: Postman
+PublisherUrl: https://www.postman.com/
+PublisherSupportUrl: https://www.postman.com/support/
+PrivacyUrl: https://www.postman.com/legal/privacy-policy/
+Author: Postman
+PackageName: Postman
+PackageUrl: https://www.postman.com/
+License: Copyright (c) 2023 Postman, Inc. All rights reserved.

```

A copyright isn't a license; 
```suggestion
License: Proprietary
```